CONTRIBUTING.md

```diff
@@ -1,165 +1,77 @@
 # Contributing
 
-We welcome feedback and help with Psammead. By participating in Psammead, you agree to abide by the [Code of Conduct](https://github.com/bbc/psammead/blob/latest/CODE_OF_CONDUCT.md). Please take a moment to read it.
-
-We appreciate all forms of contribution - not just code. Contribution can include adding documentation, requesting clarification, making typo corrections and much more.
-
-Psammead has a "Definition of Done" which applies to changes proposed to its main branch, `latest`. When you make a pull request, please check that your contribution matches this.
-
-## Definition of Done
-
-**Component Structure**
-
-As discussed in the [README](https://github.com/bbc/psammead/blob/latest/README.md), Psammead is a library of shared components following the [container / components pattern](https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0). Whether you are creating a component from scratch, or preparing to move an existing React component to Psammead, make sure that your component is presentational and reusable. The component should accept props if needed and implement minimal logic. A presentational component shouldn’t handle either fetching or mutation of data.
-
-A component should have this file structure:
-
-```
-├── README.md
-├── CHANGELOG.md
-├── package.json
-├── package-lock.json
-├── src
-│   ├── __snapshots__
-│   │   └── index.test.jsx.snap
-│   ├── index.jsx
-│   ├── index.stories.jsx
-│   └── index.test.jsx
-```
+## Documentation index
+Please familiarise yourself with our:
+- [Code of conduct](https://github.com/bbc/psammead/blob/latest/CODE_OF_CONDUCT.md)
+- [Code Standards and Ways of Working](https://github.com/bbc/simorgh/blob/latest/Code-Standards-and-Ways-of-Workinnng.md)
```

Comment:
Typo. `https://github.com/bbc/simorgh/blob/latest/Code-Standards-and-Ways-of-Working.md`

Reply:
@mykeels done